src/utils/textParse.js

```diff
@@ -32,11 +34,18 @@ function parseMentions(text, parameters) {
 	if (Object.keys(parameters).some(key => key.startsWith('mention'))) {
 		for (const [key, value] of Object.entries(parameters)) {
 			let mention = ''
-			if (value?.type === 'call') {
+
+			if (key.startsWith('mention-call') && value?.type === 'call') {
 				mention = '@all'
-			} else if (value?.type === 'user') {
+			} else if (key.startsWith('mention-federated-user') && value?.type === 'user') {
+				const server = value.server ?? getBaseUrl().split('//').pop()
```

Comment:
`pop()` sounds like a mistake. Let's speak about it tomorrow

Reply:
The url can contain the server of the hosting party so not the one where the user is coming from, right ?

Reply:
yes, but we need to omit `https` and keep `http`

Reply:
yeah just talked in the office, we need to keep the protocol if it's not `https`

Reply:
so as I understood, for {mention-federated-user} and server key does not exists, it means that the user is from the hosting server ? This means that every mention in a federated room is like {mention-federated-user}.

Reply:
> for {mention-federated-user} and server key does not exists

it means the user is from your server which is a proxy server and **not** the host of the conversation

Reply:
`{mention-federated-user}` is just a placeholder and the name pattern is "agreed on", but it could be any random sting that is `a-z0-9-`